File: 백준/Bronze/13458. 시험 감독/시험 감독.py
n = int(input())
arr = list(map(int,input().split()))
b,c = map(int,input().split())

answer = 0
for a in arr:
    a -= b
    answer += 1
    if a>0:
        if a%c == 0:
            answer += a//c
        else:
            answer += (a//c)+1

print(answer)

File: 백준/Bronze/13458. 시험 감독/시험 감독.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st;

        int N = Integer.parseInt(br.readLine());

        int[] arr = new int[N];
        st = new StringTokenizer(br.readLine());
        for (int i=0;i<N;i++){
            arr[i] = Integer.parseInt(st.nextToken());
        }

        st = new StringTokenizer(br.readLine());
        int B = Integer.parseInt(st.nextToken());
        int C = Integer.parseInt(st.nextToken());

        long answer = 0;
        for (int a : arr){
            a -= B;
            answer += 1;
            if(a>0){
                if (a%C == 0) answer += a/C;
                else answer += a/C + 1;
            }

        }

        System.out.print(answer);
    }
}